app/views/proposals/new.html.slim

```diff
@@ -0,0 +1,11 @@
+h1 Propose a Deputy Director
+= render partial: 'shared/flash_messages'
+= form_tag pq_proposal_path do |f|
+  fieldset
+    .form-group
+      - label_text = 'Deputy Director(s)'
+      = label :action_officers_pqs, :action_officer_id, label_text, value: @pq.try(:id), class: 'form-label'
+      = collection_select(:proposal_form, :action_officer_id, @action_officers, :id, :name_with_div, {include_blank: true, selected: @pq.action_officers.map(&:id)}, {multiple: true, class: 'multi-select-action-officers form-control required-for-commission', id: "action_officers_pqs_action_officer_id_#{@pq.try(:id)}" })
```

Comment:
Please replace with regular Multi-select dropdown as instructed by User Vision (Accessibility audit) - or if nothing available we can do it as a separate ticket for both dropdowns on dashboard and this new one.

Reply:
Accessibility issue but not a blocker